torchgeo/datasets/biomassters.py

```diff
@@ -74,6 +78,10 @@ def __init__(
                 Sentinel 2 ('S1', 'S2')
             as_time_series: whether or not to return all available
                 time-steps or just a single one for a given target location
+            metadata_filename: metadata file to be used
+            max_cloud_percentage: maximum allowed cloud percentage for images
+            max_red_mean: maximum allowed red_mean value for images
+            include_corrupt: whether to include images marked as corrupted
 
```

Comment:
Depending on what we decide, we need to add something like the following to the docstring:
```rst
.. versionadded:: 0.7
   The *metadata_filename*, *max_cloud_percentage*, ... parameters.
```

Reply:
thank you for the reply. I agree with the filtering on the data loader, but I didn't find any datamodule for this one.

Reply:
thank you for the reply. I agree with the filtering on the data loader, but I didn't find any datamodule for this one.